packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -265,9 +278,13 @@ export default {
     VueRecaptcha,
     CompanyLogos: () => import("../components/CompanyLogos"),
     SmartLink: () => import("../components/SmartLink"),
+    TermsAndConditionsDialog: () =>
+      import("../components/TermsAndConditionsDialog")
   },
   data() {
     return {
+      isShowTermsConditionsDialog: false,
+      socialUrl: String,
```

Comment:
Is this right? This should be a concrete value, not a data type based on the rest of the fields around it.

Reply:
`socialUrl`  is undefined until a user clicks on signup with  google/github .

Reply:
Should it just be an empty string then?
```suggestion
      socialUrl: '',
```

Reply:
Make sure to address this before merge